file.go

```diff
@@ -166,20 +166,39 @@ func (f *File) UnmarshalJSON(p []byte) error {
 	return errors.New("json struct tag unmarshal is deprecated, use ach.FileFromJSON instead")
 }
 
+// ACHFileToJSON returns an ACH file in JSON format from ACH.File.
+func (f *File) ACHFileToJSON() ([]byte, error) {
```

Comment:
I don't see the need for this method or the wordy naming. Let's call it `ToJSON` with the following body:

```Go
func (f *File) ToJSON() ([]byte, error) {
	return json.Marshal(f)
}
```